docs/install/pxe-boot-install.md

```diff
@@ -18,7 +18,7 @@ Starting from version `0.2.0`, Harvester can be installed in a mass manner. This
 
 We recommend using [iPXE](https://ipxe.org/) to perform the network boot. It has more features than the traditional PXE Boot program and is likely available in modern NIC cards. If NIC cards don't come with iPXE firmware, iPXE firmware images can be loaded from the TFTP server first.
 
-To see sample iPXE scripts, please visit https://github.com/harvester/ipxe-examples.
+To see sample iPXE scripts, please visit [Harvester iPXE Examples](https://github.com/harvester/ipxe-examples).
```

Comment:
add a hyperlinks in 'pxe install' page because the original one is displayed as plain texts.